components/match2/wikis/wildrift/get_match_group_copy_paste_wiki.lua

```diff
@@ -0,0 +1,106 @@
+---
+-- @Liquipedia
+-- wiki=wildrift
+-- page=Module:GetMatchGroupCopyPaste/wiki
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Table = require('Module:Table')
+
+--[[
+
+WikiSpecific Code for MatchList and Bracket Code Generators
+
+]]--
+
+local wikiCopyPaste = Table.copy(require('Module:GetMatchGroupCopyPaste/wiki/Base'))
+
+--allowed opponent types on the wiki
+local MODES = {
+	['solo'] = 'solo',
+	['team'] = 'team',
+	['literal'] = 'literal',
+}
+
+--default opponent type (used if the entered mode is not found in the above table)
+local DefaultMode = 'team'
+
+--returns the cleaned opponent type
+function wikiCopyPaste.getMode(mode)
+	return MODES[string.lower(mode or '')] or DefaultMode
+end
```

Comment:
Unused

Reply:
used by the commons module, this one is just the custom part
> `local mode = WikiSpecific.getMode(args.mode)` in line 110 of `Module:GetMatchGroupCopyPaste`